packages/web/src/new/scenes/Main/Settings/Settings.ViewModel.ts

```diff
@@ -36,6 +40,8 @@ export class SettingsViewModel extends ViewModel {
     void this._nameService
       .getName(this._solanaService.provider.wallet.publicKey.toBase58())
       .then(action((name) => name && (this.username = name)));
+
+    this.walletModel.initialize();
```

Comment:
Looks like `this.walletModel.initialize()` is already initilized in `RootViewModel`, so we don't need it again. Correct me if i wrong